APR-Dragonflight/Dragonflight.lua

```diff
@@ -6055,108 +6056,225 @@ APR.QuestStepList["DF04-2023-OhnahranPlains"] = {
           },
      },
      {	-- Step 194
-          ["Qpart"] = {
+          ["QpartPart"] = {
                [66257] = {
                     ["1"] = "1",
                },
           },
           ["TT"] = {
-               ["y"] = -1981.7,
-               ["x"] = 2416.8,
+               ["y"] = -1982.1,
+               ["x"] = 2415.9,
           },
           ["Trigger"] = {
-               ["y"] = -1981.7,
-               ["x"] = 2416.8,
+               ["y"] = -1982.1,
+               ["x"] = 2415.9,
           },
-          ["Range"] = 500,
+          ["Range"] = 3,
           ["Fillers"] = {
                [66242] = {
                     ["1"] = "1",
                },
+               [66242] = {

```

Comment:
This won't work. It has to be 
```
[66242] = {
                    ["1"] = "1",
                    ["2"] = "2",
               },
```
Otherwise, the first objective gets ignored.